src/main/java/com/bloomtech/asylumrgbea/models/CasesQueryParameterDto.java

```diff
@@ -0,0 +1,66 @@
+package com.bloomtech.asylumrgbea.models;
+
+import lombok.Data;
+import org.springframework.lang.Nullable;
+
+import java.util.Objects;
+
+/**
+ * Defines the DTO to store the query parameters provided by the client
+ * for the cases' endpoint. Each attribute represents a query parameter
+ * or category to filter cases by.
+ */
+@Data
+public class CasesQueryParameterDto {
+    private int limit = 10;
+    private int page = 1;
+    @Nullable
+    private String citizenship;
+    @Nullable
+    private String outcome;
+    @Nullable
+    private String from;
+    @Nullable
+    private String to;
+    @Nullable
+    private String office;
+
```

Comment:
Are we using query parameters "received" or "fiscal"? Or, maybe that will be implemented in a future release?

Reply:
dateRecieved and fiscal is out of scope. However, fiscal will be implemented in a future release.